codecov.yml

```diff
@@ -1,12 +1,18 @@
 # Don't post a comment on pull requests.
-comment: off
+comment: false
 
 # Disable commit statuses
 coverage:
   status:
-    project: no
-    patch: no
-    changes: no
+    project:
```

Comment:
The [docs](https://docs.codecov.io/v4.3.6/docs/commit-status) provide the following example:

```yaml
coverage:
  status:
    project:
      default: false  # disable the default status that measures entire project
```

I can't find any docs that use the format `enabled: no`, can you point me to where you found an example that this config is following?

Reply:
@mbrukman I'll try `default: false`, that's still different than what was there and the docs look more current. I was looking at https://github.com/codecov/support/wiki/Codecov-Yaml#full-yaml.